src/utils/splitHiddenTokens.tsx

```diff
@@ -1,29 +1,26 @@
-import { TokenBalance } from 'graphql/data/__generated__/types-and-hooks'
+import { TokenBalance, TokenStandard } from 'graphql/data/__generated__/types-and-hooks'
 
 const HIDE_SMALL_USD_BALANCES_THRESHOLD = 1
 
 export function splitHiddenTokens(
   tokenBalances: TokenBalance[],
-  options?: {
+  options: {
     hideSmallBalances?: boolean
-  }
+  } = { hideSmallBalances: true }
 ) {
   const visibleTokens: TokenBalance[] = []
   const hiddenTokens: TokenBalance[] = []
 
   for (const tokenBalance of tokenBalances) {
-    const isValidValue =
+    const isUndefinedValue =
       // if undefined we keep visible (see https://linear.app/uniswap/issue/WEB-1940/[mp]-update-how-we-handle-what-goes-in-hidden-token-section-of-mini)
```

Comment:
i think this comment should go above line 15 now? or else the line break here is very awk